src/fixture/smoke/login.fixture.ts

```diff
@@ -1,7 +1,6 @@
-import routes from '../../utils/routes.utils';
-
 export default {
   username: 'robocop@mail.test',
-  password: 'robocop12345',
-  url: routes.projects
+  password: 'robocop1234532132',
+  invalidUsername: '123',
+  invalidPassword: '123'
```

Comment:
Why you create two params with the same values? Rename it like `invalidData` then use this param in test